frameworks/kafka/tests/test_overlay.py

```diff
@@ -83,7 +83,7 @@ def test_pods_restart_on_overlay():
 @overlay_nostrict
 @sdk_utils.dcos_1_9_or_higher
 def test_pods_replace_on_overlay():
```

Comment:
Given that the test only replaces one broker pod, the name and the function being called should be changed.